doc/modules/model_evaluation.rst

```diff
@@ -1551,6 +1552,32 @@ Here is a small example of usage of the :func:`mean_absolute_error` function::
   ... # doctest: +ELLIPSIS
   0.849...
 
+.. _mean_absolute_percentage_error:
+
+Mean absolute error
+-------------------
+
+The :func:`mean_absolute_percentage_error` function computes `mean absolute
+percentage error <https://en.wikipedia.org/wiki/Mean_absolute_percentage_error>`_, a risk
+metric corresponding to the expected value of the absolute percentage error loss or
+:math:`l1`-norm of percentage loss.
+
+If :math:`\hat{y}_i` is the predicted value of the :math:`i`-th sample,
+and :math:`y_i` is the corresponding true value, then the mean absolute percentage error
+(MAPE) estimated over :math:`n_{\text{samples}}` is defined as
```

Comment:
I would put the (MAPE) at the first mention of mean absolute percentage error above.

Also maybe add at least one sentence of explanation, say
"MAPE computes the error relative to the true value. Therefore the same absolute distance between prediction and ground truth will lead to a smaller error if the true value is larger. In particular the metrics is not shift-invariant.

Reply:
done